src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/FullscreenTextResource.svelte

```diff
@@ -119,7 +133,7 @@
                     </div>
                 {/if}
                 <div
-                    class="prose mx-2 rounded-md {resource?.communityEdition &&
+                    class="prose mx-auto w-full rounded-md {resource?.communityEdition &&
```

Comment:
This fixes a bug where content with a narrow width would get centered and look odd.